### ISSUE TITLE
Shape of predictions

### ISSUE THREAD

**Issue by Marcosuff** [deepchem/deepchem#1606]
Hello,
I just trained my first model using Deepchem but something seems odd when I call mode.predict(). It does return a numpy array of predictions but in a different shape to my data.
My train_dataset has shape (4784,5) while my train_predictions has shape (4784,5,2). 
Why is it a 3d array? Here's a small sample of my code.

    metric = dc.metrics.Metric(dc.metrics.roc_auc_score, np.mean, mode='classification')

    model = dc.models.MultitaskClassifier(n_features=train_dataset.get_data_shape()[0],
    n_tasks=len(tasks),layer_sizes=[500],dropouts=[.25],
    optimizer='adam',learning_rate=1e-3,batch_size=64)

    # Check shape of y_true
    print(train_dataset.y.shape)
    ### (4784,5)

    print("Training model")
    ### Training model

    model.fit(train_dataset,nb_epoch=60)

    # Make predictions

    preds = model.predict(train_dataset)
   
    # Check predictions shape
    print(preds.shape)
    ### (4784,5,2)

**Comment by peastman** [comment]
The output of MultitaskClassifier has shape (n_samples, n_tasks, n_classes).

**Comment by Marcosuff** [comment]
> The output of MultitaskClassifier has shape (n_samples, n_tasks, n_classes).

Thanks @peastman. So the slice **predictions[:,:,1]** corresponds to the probabilities for class 1?

**Comment by peastman** [comment]
Right.
